### ISSUE TITLE
Expose extra info to mutate callback

### ISSUE THREAD

**Issue by markhughes** [shellscape/apollo-log#11]
### Feature Use Case

Expose the context (`requestContext) in the `mutate` callback, this way we can grab HTTP, misc. context, headers, operationName, variables, etc, and log extra breadcrumbs (such as a session id from headers) to log as a single key for logging systems. 

### Feature Proposal

All implementation like so:

```diff
--      mutate: (data) => {
++      mutate: (data, { http, operationName }) => {
```

This way we can cherry pick the variables we need from context with our extra logging requirements!

**Comment by shellscape** [comment]
Sure. Create a pull request and we'll take a look.

**Comment by markhughes** [comment]
Thanks, I've opened https://github.com/shellscape/apollo-log/pull/12 for it now. :)

**Comment by shellscape** [comment]
Closed by #12

**Comment by markhughes** [comment]
Thank you! I think this will benefit a lot of people.
